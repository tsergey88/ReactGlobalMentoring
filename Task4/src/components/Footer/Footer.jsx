import React from "react";

export class Footer extends React.Component {
  render() {
    return (
      <footer>
        <div>NETFLIX ROULETTE</div>
      </footer>
    );
  }
}
