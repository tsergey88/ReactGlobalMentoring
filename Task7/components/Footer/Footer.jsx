import React from "react";

export const Footer = () => (
  <footer>
    <div>NETFLIX ROULETTE</div>
  </footer>
);

export default Footer;
