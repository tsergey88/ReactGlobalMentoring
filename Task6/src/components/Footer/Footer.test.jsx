import React from "react";
import { Footer } from "./Footer";
import { shallow } from "enzyme";

describe("Footer component", () => {
  const component = shallow(<Footer />);

  it("should be render correctly", () => {
    expect(component).toMatchSnapshot();
  });

  it("should contain footer", () => {
    expect(component.find("footer")).toHaveLength(1);
  });

  it("should contain correct div", () => {
    expect(component.find("div")).toHaveLength(1);
  });
});
