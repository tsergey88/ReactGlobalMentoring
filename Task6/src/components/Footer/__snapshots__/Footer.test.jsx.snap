// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should be render correctly 1`] = `
<footer>
  <div>
    NETFLIX ROULETTE
  </div>
</footer>
`;
